docs/translations/readme.md

```diff
@@ -0,0 +1,4 @@
+<!--
```

Comment:
@marbar3778 Is there any reason for having `readme.md` in addition to `README.md`, or just an oversight? This throws Git warnings on macOS since its filesystem is case-insensitive, and these two filenames therefore map to the same file.

Reply:
I think someone pushed this on accident. I am opposed to having empty readme's but i believe they are needed for the vuepress config.